src/components/pages/dashboard-view/DashboardViewContainer.jsx

```diff
@@ -61,21 +33,21 @@ const DashboardViewContainer = () => {
       }}
       key="app-container"
     >
-      {fakeData.map(program => (
-        <GhostLink to={VIEW_PROGRAM_PATH}>
+      {programs.map(program => (
+        <GhostLink to={`/program/${program[0].programId}`}>
```

Comment:
A little confused as to why our `program` property is an array... shouldn't this be an object? This may allude to the fact that we should tweak the shape of our `state.programs` slice

Reply:
Also, it might make sense to split out our `<GhostLink><Card><Meta>...<///>` into a separate `<ProgramCard />` component

Reply:
Yes totally agree. Let's circle back on this once this is pushed through!